docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1156 @@
+----------
+*Back to* `Index`_
+
+The role of this sub-module was to convert the operators into other opflow operator classes
+(:class:`~qiskit.opflow.converters.TwoQubitReduction`, :class:`~qiskit.opflow.converters.PauliBasisChange`...).
+In the case of the :class:`~qiskit.opflow.converters.CircuitSampler`, it traversed an operator and outputted
+approximations of its state functions using a quantum backend.
+Notably, this functionality has been replaced by the :mod:`~qiskit.primitives`.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow
+     - Alternative
+
+   * - :class:`~qiskit.opflow.converters.CircuitSampler`
+     - :class:`~primitives.Estimator`
+   * - :class:`~qiskit.opflow.converters.AbelianGrouper`
+     - No direct replacement. This class allowed a sum a of Pauli operators to be grouped. These type of groupings are now left to the primitives to handle.
+   * - :class:`~qiskit.opflow.converters.DictToCircuitSum`
+     - No direct replacement
+   * - :class:`~qiskit.opflow.converters.PauliBasisChange`
+     - No direct replacement
+   * -  :class:`~qiskit.opflow.converters.TwoQubitReduction`
+     -  No direct replacement. This class implements a chemistry-specific reduction for the ``ParityMapper`` class in ``qiskit-nature``.
+        The general symmetry logic this mapper depends on has been refactored to other classes in :mod:`~qiskit.quantum_info`,
+        so this specific :mod:`~qiskit.opflow` implementation is no longer necessary.
+
+
+.. _convert_state:
+
+Example 1: ``CircuitSampler`` for sampling Parametrized Circuits
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+.. raw:: html
+
+    <details>
+    <summary><a><b>Opflow</b></a></summary>
+
+.. code-block:: python
+
+    from qiskit_aer import Aer
+    from qiskit.circuit import QuantumCircuit, Parameter
+    from qiskit.opflow import ListOp, StateFn, CircuitSampler
+
+    x, y = Parameter("x"), Parameter("y")
+
+    circuit1 = QuantumCircuit(1)
+    circuit1.p(0.2, 0)
+    circuit2 = QuantumCircuit(1)
+    circuit2.p(x, 0)
+    circuit3 = QuantumCircuit(1)
+    circuit3.p(y, 0)
+
+    bindings = {x: -0.4, y: 0.4}
+    listop = ListOp([StateFn(circuit) for circuit in [circuit1, circuit2, circuit3]])
+
+    sampler = CircuitSampler(Aer.get_backend("aer_simulator"))
+    sampled = sampler.convert(listop, params=bindings).eval()
+    # returns list of SparseVectorStateFn
+
+.. raw:: html
+
+    </details>
+
+.. raw:: html
+
+    <details>
+    <summary><a><b>Alternative</b></a></summary>
+
+.. code-block:: python
+
+    from qiskit.circuit import QuantumCircuit, Parameter
+    from qiskit.primitives import Sampler
+
+    x, y = Parameter("x"), Parameter("y")
+
+    circuit1 = QuantumCircuit(1)
+    circuit1.p(0.2, 0)
+    circuit2 = QuantumCircuit(1)
+    circuit2.p(x, 0)
+    circuit3 = QuantumCircuit(1)
+    circuit3.p(y, 0)
+
+    circuits = [circuit1, circuit2, circuit3]
+    param_values = [[-0.4, 0.4] for _ in circuits]
```

Comment:
Would it be better to do only give the actual parameter values for each circuit?
```suggestion
    param_values = [None, [-0.4], [0.4]]
```